.github/workflows/codeql.yml

```diff
@@ -1,10 +1,10 @@
 name: "CodeQL"
 
 on:
-  push:
-    branches: [ "master", "develop", "release-*" ]
```

Comment:
We are currently running this check twice on every merge commit.

Reply:
good catch